pose-detection/src/calculators/interfaces/common_interfaces.ts

```diff
@@ -43,3 +44,12 @@ export interface KeypointsFilter {
       Keypoint[];
   reset(): void;
 }
+
+export interface Track {
+  id: number;  // A unique identifier for each tracker person.
+  lastTimestamp: number;  // The last timestamp (in milliseconds) in which a
+                          // detection was linked with the track.
+  keypoints?: Keypoint[];  // The most recent set of keypoints.
```

Comment:
Do you want to force trackers to use only the most recent set of keypoints and bounding box? I guess a slightly more advanced tracker would want to keep some history to make predictions of where it expects keypoints and bounding boxes.